release_notes/v2.4.0.md

```diff
@@ -16,27 +16,78 @@ The Fabric Gateway will also simplify the administrative overhead of running a F
 will be able to connect and submit transactions via a single network port rather than the current situation where ports
 have to be opened from a client application to multiple peers across potentially multiple organizations.
 
-The Fabric Gateway is delivered along with slim SDKs in the [https://github.com/hyperledger/fabric-gateway](https://github.com/hyperledger/fabric-gateway) repository,
-along with [client application samples](https://github.com/hyperledger/fabric-gateway/tree/main/samples).
+The Fabric Gateway is delivered along with slim SDKs in the [https://github.com/hyperledger/fabric-gateway](https://github.com/hyperledger/fabric-gateway) repository.
+Check out the [client application samples](https://github.com/hyperledger/fabric-gateway/tree/main/samples).
 
+**[FAB-11334] Unjoin a channel from a peer**
+
+The new command `peer node unjoin` enables an administrator to remove (unjoin) a channel from a peer.
+The peer must be stopped when the command is executed so that channel artifacts can be cleaned up.
+The channel's blockchain, state database, and associated entries will be removed from the peer.
+When the peer is restarted it will no longer receive blocks for the channel.
 
 Improvements
 ------------
 
-TBD
+**peer and orderer - Implement legacy name constraints verification for Go 1.15**
```

Comment:
This is not really an improvement though. We just implemented code that forces the legacy behavior, to preserve consistency. Not sure why this is mentioned in the release notes

Reply:
I could go either way, but I thought it would be nice to have it there for the record along with the upgrade to Go 1.15/1.16 in the release notes, in case anybody had concerns about the change in Go behavior.